### DISCUSSION TITLE
10/24 김혜빈 블로그 생성

### DISCUSSION THREAD

**Discussion by hyebinkimsdf** [Orm-camp/blog-feedback#26] [동글동글 3기 피드백]
이곳에 처음 와서 html을 하면서 과제 말고 개인작으로 처음 만드는 것 같아요!
저는 직접 디자인하고 직접 개발하여 포트폴리오용 홈페이지를 만들고 싶습니다.
공부했던 내용도 올리는 페이지도 따로 만들어서 올리겠습니다!

아직 미숙하지만 ㅠㅠ 열심히 해보고 싶어요! 다른 분들도 파이팅입니다!

**Comment by hyebinkimsdf** [comment]
https://hyebinkimsdf.github.io/hyebinkim.github.io/ 입니다

**Comment by ksj0621** [comment]
저도 같이 시작하는 초심자라 함께 힘내보고 싶습니다, 블로그 종종 들르겠습니다:)

**Comment by ghost** [comment]
직접 만드시다니 멋지시네요!
응원하겠습니다~
